src/test/java/com/devcycle/sdk/server/local/LocalBucketingTest.java

```diff
@@ -75,7 +75,7 @@ public void testStoreConfig() {
     public void testEventQueue() throws JsonProcessingException {
         DevCycleEvent event = DevCycleEvent.builder().type("test").target("target").build();
 
-        localBucketing.initEventQueue(apiKey, "{}");
+        localBucketing.initEventQueue(apiKey, UUID.randomUUID().toString(),"{}");
```

Comment:
you'll need to track this client uuid attaching it to the localbucketing instance - as it's used later for sdkConfig events

Reply:
I'd rather get that done in the same pr as the sdkConfig if it hasn't been done already

Reply:
it has - just an fyi